bedrock/mozorg/templates/mozorg/about-en.html

```diff
@@ -71,16 +71,29 @@ <h1 class="c-page-title">{{ _('Our Mission in Action') }}</h1>
           aspect_ratio='mzp-has-aspect-3-2',
           link_url='https://blog.mozilla.org/Internetcitizen/2018/06/06/the-today-show-mozilla-firefox-facebook-container/'
       )}}
+      {% if l10n_has_tag('irl_card_about_2018') %}
```

Comment:
Looks like this page hasn't been exposed for l01n yet so we don't need an l10n tag after all, you can just update the card with no fallback.